doc/man7/provider-asym_cipher.pod

```diff
@@ -241,7 +241,7 @@ decryption. When set (non zero value), the decryption API will return
 a deterministically random value if the PKCS#1 v1.5 padding check fails.
 This makes exploitation of the Bleichenbacher significantly harder, even
 if the code using the RSA decryption API is not implemented in side-channel
-free manner. Set by default.
+free manner. Set by default. Requires provider support.
```

Comment:
I think the part "Requires provider support" can be more clear. Perhaps do you want to say here if any providers except the "default" provider requires an similar implementation like "implicit-rejection"?

Reply:
I know that the ibmca provider will use implicit rejection by default, I'm not sure if it's possible to disable that behaviour, using this flag or any other.
`default` provider is the main provider that can use it, but there may be others.